runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -98,21 +94,9 @@
 
 
   class JobNameFactory implements DefaultValueFactory<String> {
```

Comment:
used for `JobName` above, but is that actually still needed? or can they use `getNormalizedUniqueName()` instead?